bvm/ballerina-core/src/main/java/org/ballerinalang/util/debugger/DebugInfoHolder.java

```diff
@@ -85,7 +85,7 @@ private void processPkgInfo(PackageInfo packageInfo) {
 
     private void addDebugPoint(BreakPointDTO breakPointDTO) {
         if (packageInfoMap.get(breakPointDTO.getPackagePath()) == null) {
-            return;
+            throw new DebugException(DebugConstants.MSG_INVALID_BREAKPOINT + breakPointDTO.toString());
```

Comment:
@Shan1024 It's being handled at the [edge](https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-core/src/main/java/org/ballerinalang/util/debugger/Debugger.java#L136) and consistent with [other places](https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-core/src/main/java/org/ballerinalang/util/debugger/Debugger.java#L260) that does the same. EOD it will send a  INVALID_CODE type message and debugger will be stopped.

Reply:
I think we should not stop the debugger because of trying to register an invalid breakpoint. We should simply return a error message and continue the debugger. The other place you have mentioned, debugger cannot continue since the worker context is `null`. So I think stopping the debugger is the correct behavior in that case. 

@lankavitharana - WDYT?

Reply:
+1 Agreed. Let's change it to send an error message and allow continuing to debug.

Reply:
yes we need to have two parts, like there may be cases that debugger only needs to inform the client about the error, but debugger can continue, and there are cases  that debugger cannot continue, +1 this doesn't need to stop debugger, rather inform the client about wrong debug line and continue debugging. I think for most scenarios we can do that.